src/components/PermissionIndicator/PermissionIndicator.tsx

```diff
@@ -13,20 +13,22 @@ import {
 } from '@tabler/icons-react';
 import React from 'react';
 
+const canSellImagesPermissions = ['Image', 'Rent', 'RentCivit', 'Sell'] as const;
+const canRentCivitPermissions = ['RentCivit', 'Rent', 'Sell'] as const;
+const canRentPermissions = ['Rent', 'Sell'] as const;
+const canSellPermissions = ['Sell'] as const;
```

Comment:
Curious why you're not using the Prisma enum values here

Reply:
Just to save a few key strokes 😛